MIGRATION.md

```diff
@@ -1,5 +1,32 @@
 # Migration guides
 
+## 3.3.x to 3.4.x
+
+- `foregroundService` is no longer available in `RadarTrackingOptions`. This has been replaced by `Radar.setForegroundServiceOptions` instead.
```

Comment:
@jsani-radar you're in the loop with this and cool with it?

Reply:
yeah this was discussed with @jkao way back - basically don't want to conflate options that belong on the client (ex. fg service icon) with clunkily specifying those in the dashboard